src/App.vue

```diff
@@ -4,11 +4,17 @@
 </template>
 
 <style>
+@import url('https://fonts.googleapis.com/css2?family=Forum&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+
+body {
+  font-size: calc(14px + (26 - 14) * ((100vw - 300px) / (1600 - 300)));
+}
+
```

Comment:
Let's use Google font Open Sans instead throughout the application

Reply:
![image](https://user-images.githubusercontent.com/59277035/117961316-9a69cf00-b33b-11eb-95b5-dacd14cc171c.png)
For logo purpose i used that